<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2500</width>
    <height>1400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ЛР 2. Зайцева (Вариант 8)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>681</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры генератора (экспоненциальный закон)</string>
    </property>
    <widget class="QDoubleSpinBox" name="gen_int_min">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Интенсивность (x1)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>max:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>min:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="gen_int_max">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>641</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры ОА (нормальный закон)</string>
    </property>
    <widget class="QDoubleSpinBox" name="proc_int_min">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>4.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Интенсивность (x2)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Разброс интенсивности (x3)</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="proc_var_min">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.001000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>min:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="proc_var_max">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>100</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>max:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>min:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="proc_int_max">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>max:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_model">
    <property name="geometry">
     <rect>
      <x>1950</x>
      <y>30</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>1150</y>
      <width>761</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Результаты</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Загрузка системы</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Теоретически</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>40</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Экспериментально</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="res_theor_zagr">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>90</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="res_fact_zagr">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>90</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="res_exp_mean_time_in_queue">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ср. время ожидания в очереди</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="res_theor_mean_time_in_queue">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>20</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Окончание моделирования</string>
    </property>
    <widget class="QDoubleSpinBox" name="input_t">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="value">
      <double>10000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество заявок</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>2400</width>
      <height>600</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>13</number>
    </property>
    <column>
     <property name="text">
      <string>x0</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x1x2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x1x3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x2x3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x1x2x3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>y</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>yл</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>yнл</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>|y-yлl</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>|y-yнл|</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>1070</y>
      <width>681</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Проверка в точке (кодированные значения)</string>
    </property>
    <widget class="QDoubleSpinBox" name="input_intens_generator">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Интенсивность генератора (x1)</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="input_disp_processor">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Разброс интенсивности ОА (x3)</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="input_intens_processor">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Интенсивность ОА (x2)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="check_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Проверить</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>850</y>
      <width>2111</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Регрессионные зависимости</string>
    </property>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Линейная</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Частично нелинейная</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="linear">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>1500</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="nonlinear">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>1500</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
